model/simulationtests/autosize_hvac.rb

```diff
@@ -831,6 +910,20 @@ def curve_quadratic(model, c_1constant, c_2x, c_3xPOW2, minx, maxx, miny, maxy)
 humidistat.setHumidifyingRelativeHumiditySetpointSchedule(dehumidify_sch)
 zones[40].setZoneControlHumidistat(humidistat)
 
+# TES Coil CoilCoolingDXSingleSpeedThermalStorage
+tes_air_loop_unitary = OpenStudio::Model::AirLoopHVACUnitarySystem.new(model)
+tes_coil = make_tes_coil(model)
+fan = OpenStudio::Model::FanOnOff.new(model)
+tes_air_loop_unitary.setAvailabilitySchedule(model.alwaysOnDiscreteSchedule)
+tes_air_loop_unitary.setCoolingCoil(tes_coil)
+tes_air_loop_unitary.setSupplyFan(fan)
+tes_air_loop = OpenStudio::Model::AirLoopHVAC.new(model)
+tes_air_loop_unitary.addToNode(tes_air_loop.supplyInletNode)
+tes_air_loop_unitary.setControllingZoneorThermostatLocation(zones[43])
+tes_atu = OpenStudio::Model::AirTerminalSingleDuctConstantVolumeNoReheat.new(model, s1)
+tes_air_loop.addBranchForZone(zones[43], tes_atu)
+tes_air_loop.setName('AirLoop with TES Coil')
```

Comment:
Renamed the variables to be tes_ prefixed so I don't steal others defined before and normally used after (air_loop in particular).